test/source/tests/settings.ts

```diff
@@ -288,13 +288,9 @@ export let defineSettingsTests = (testVariant: TestVariant, testWithBrowser: Tes
       await myKeyFrame.notPresent('@action-update-prv');
       await myKeyFrame.notPresent('@action-revoke-certificate');
       await settingsPage.notPresent('@action-remove-key');
-      const fingerprint = await myKeyFrame.readHtml('@content-fingerprint');
-      // test for direct access at my_key_update.htm
-      const myKeyUpdateFrame = await browser.newPage(t, TestUrls.extension(`chrome/settings/modules/my_key_update.htm?placement=settings&acctEmail=${acct}&fingerprint=${fingerprint}`));
-      await myKeyUpdateFrame.waitForContent('@container-err-title','Error: Insufficient Permission');
```

Comment:
Instead of removing test for title, please test for the message / comment itself that is rendered in this situation. Thank you! Very sorry I broke this PR during the merge.

Reply:
Done 👍